src/test/java/com/wikia/webdriver/testcases/flagTests/flagTest.java

```diff
@@ -0,0 +1,47 @@
+package com.wikia.webdriver.testcases.flagtests;
```

Comment:
see the folder name. You named it flagTests with cammel case. It creates compilation error. Change flagTests path to flagtests